packages/ui/src/partials/button.tsx

```diff
@@ -3,16 +3,20 @@ import { Slot } from "@radix-ui/react-slot";
 import { cva, type VariantProps } from "class-variance-authority";
 
 import { cn } from "../../src/lib/tw-class-merge";
+import { LucideIcon } from "lucide-react";
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-white transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",
+  "inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium ring-offset-white transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50",
   {
     variants: {
       variant: {
         default: "grow bg-black text-white hover:bg-black/80",
+        provider: "border text-black hover:bg-black/5",
+        link: "text-black hover:text-black/80",
```

Comment:
```suggestion
        link: "text-black hover:text-black/80",
```
will the variant type link be needed for button?